samples/Kconfig

```diff
@@ -101,9 +101,11 @@ config LOG_DEFAULT_LEVEL
 endif # LOG
 
 config NCS_IS_VARIANT_IMAGE
-	bool "Image is a variant build of another image [READ ONLY]"
+	bool
+	help
+	   Image is a variant build of another image.
 
 config NCS_MCUBOOT_IN_BUILD
-	bool "MCUBoot is included in the build [READ ONLY]"
+	bool
```

Comment:
this is set by CMake on the child image here:
https://github.com/nrfconnect/sdk-nrf/blob/51167148f0f2158096e9fb54a02337f6f3737a22/cmake/multi_image.cmake#L225

For this to work, this setting needs a prompt, else CMake cannot set it.
CMake is the "user" in this case.

Reply:
Thank you, I opted for putting them into their own menu so they no longer mix with other top-level menu entries.